SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -301,6 +317,32 @@ void FLocalDeploymentManager::TryStartLocalDeployment(FString LaunchConfig, FStr
 	return;
 }
 
+bool FLocalDeploymentManager::SetupRuntimeFileLogger(FString RuntimeLogDir)
+{
+	// Ensure any old log file is cleaned up.
+	RuntimeLogFileHandle.Reset();
+
+	FString RuntimeLogFilePath = FPaths::Combine(RuntimeLogDir, TEXT("runtime.log"));
+	IPlatformFile& PlatformFile = FPlatformFileManager::Get().GetPlatformFile();
+
+	const bool bSuccess = PlatformFile.CreateDirectoryTree(*RuntimeLogDir);
+
+	if (bSuccess)
+	{
+		RuntimeLogFileHandle.Reset(PlatformFile.OpenWrite(*RuntimeLogFilePath, /*bAppend*/ true, /*bAllowRead*/ true));
```

Comment:
Are either of these needed? 
```suggestion
		RuntimeLogFileHandle.Reset(PlatformFile.OpenWrite(*RuntimeLogFilePath, /*bAppend*/ false, /*bAllowRead*/ false));
```

Reply:
Maybe not bAppend, but my thought is that bAllowRead allows a user / other thread to read the file while we're accessing it, so it's not locked